src/app/asset-upload/asset-upload.component.html

```diff
@@ -5,7 +5,8 @@
                 Asset Upload
             </mat-panel-title>
         </mat-expansion-panel-header>
-        <button mat-raised-button 
+        <div class="button-filename">
+            <button mat-raised-button 
             color="primary" 
             type="button" 
             (click)="fileInput.click()">
```

Comment:
Align all the element attributes. Examples of good formatting here: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Indent child elements: https://google.github.io/styleguide/htmlcssguide.html#General_Formatting